*** compile/config_default.mk.orig	Mon Jan 24 01:41:56 2005
--- compile/config_default.mk	Mon Jan 24 01:42:44 2005
***************
*** 35,45 ****
  PERL		= perl
  BUNZIP2		= bzip2 -d
  GUNZIP		= gzip -d
! FILTER_CP932	= qkc -q -u -s
! FILTER_EUCJP	= qkc -q -u -e
! FILTER_UTF8	= iconv -t utf-8 -f cp932
! #FILTER_CP932	= nkf -s
! #FILTER_EUCJP	= nkf -e
  INSTALL		= /usr/bin/install -c
  #INSTALL	= /usr/ucb/install -c
  INSTALL_PROGRAM	= $(INSTALL) -m 755
--- 35,43 ----
  PERL		= perl
  BUNZIP2		= bzip2 -d
  GUNZIP		= gzip -d
! FILTER_CP932	= nkf -s
! FILTER_EUCJP	= nkf -e
! FILTER_UTF8		= nkf -w
  INSTALL		= /usr/bin/install -c
  #INSTALL	= /usr/ucb/install -c
  INSTALL_PROGRAM	= $(INSTALL) -m 755
